felix/fv/winfv/policy_test.go

```diff
@@ -178,13 +216,26 @@ var _ = Describe("Windows policy test", func() {
 					},
 				},
 			}
-			_, err := client.NetworkPolicies().Create(context.Background(), &p, options.SetOptions{})
+			_, err = client.NetworkPolicies().Create(context.Background(), &p, options.SetOptions{})
 			Expect(err).NotTo(HaveOccurred())
 			defer func() {
 				_, err = client.NetworkPolicies().Delete(context.Background(), "demo", "allow-nginx-b", options.DeleteOptions{})
 				Expect(err).NotTo(HaveOccurred())
 			}()
 
+			By("asserting destination is not reachable")
+			// Assert nginx-b is not reachable.
+			kubectlExecWithErrors(fmt.Sprintf(`-t porter -- powershell -Command 'Invoke-WebRequest -UseBasicParsing -TimeoutSec 5 %v'`, nginxB))
+
+			By("updating tier1 default action to pass")
+			tier1, err = client.Tiers().Get(context.Background(), tier1.Name, options.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			passAction := v3.Pass
+			tier1.Spec.DefaultAction = &passAction
```

Comment:
Suggest you add the same default pass rule to ingress policy tests as well.

Reply:
@song-jiang do you suggest to add a new test for ingress? Cause I don't see any other existing test.

Also, the tier default action works the same for both ingress and egress. And I tried to cover all combinations in the UTs.

Reply:
Agree egress policy test probably is good enough given we have UTs to cover all combinations.